src/app/reactUtils.spec.tsx

```diff
@@ -81,6 +82,46 @@ describe('useFocusLost', () => {
   });
 });
 
+describe('useTimeout', () => {
+  // tslint:disable-next-line:variable-name
+  let Component: React.ComponentType;
+  let callback: jest.Mock;
+  beforeEach(() => {
+    resetModules();
+    jest.useFakeTimers();
+
+    callback = jest.fn();
+
+    Component = () => {
+      const [delay, setDelay] = React.useState(1000);
+      utils.useTimeout(delay, callback, []);
+
+      return <button onClick={() => setDelay(2000)}></button>;
+    };
+  });
```

Comment:
@PiotrKozlowski what do you think about testing hooks in an actual component and not extracting their body into handlers? The hook allows the `delay` to change but there is no a component that does that (yet?), which lead me to creating a small test component that was doing just that.

Reply:
I think that's okay. You could also add tests for this hook in tests for `SearchFailure` component where this hook is used

Reply:
Well all of its other functionality (preserving the timeout, calling callback and etc.) is actually tested there. Just didnt have a way of testing the delay change.

Reply:
oh, then I think it's totally fine to create this test component like you did